tests/app/submitter/test_submitter.py

```diff
@@ -336,3 +338,58 @@ def test_gcs_feedback_submitter_uploads_feedback(patch_gcs_client):
         b'"form_type": "H", "language_code": "cy", "region_code": "GB-ENG", "tx_id": "12345"}'
     )
     assert feedback_upload is True
+
+
+def test_double_submission_passes(
+    patch_gcs_client, gcs_blob_storage_delete_error_message
+):  # pylint: disable=redefined-outer-name
+    # Given
+    gcs_submitter = GCSSubmitter(bucket_name="test_bucket")
+
+    # When
+    bucket = patch_gcs_client.return_value.get_bucket.return_value
+    bucket.blob.return_value = gcs_blob_storage_delete_error_message
+
+    published = gcs_submitter.send_message(
+        message={"test_data"}, tx_id="123", case_id="456"
+    )
+    # Then
+    assert published
+
+
+def test_forbidden_error_raised(
```

Comment:
I feel like the test names here could be more descriptive so that it is easier to understand the scenarios being covered, i.e something like `test_double_submission_is_forbidden_when_not_delete_operation_error`?